### PR TITLE
Docs: Include OTLP in the tracing references

### PR DESCRIPTION
**Pull request by Symbianx**
**What this PR does / why we need it**:

Changes the docs about monitoring Grafana to include OTLP as a supported tracing backend.

### CODE REVIEW

**Comment by chri2547** [comment]
@grafana/user-essentials , I'm unsure if this falls in your domain. I'd like someone to review this PR for technical accuracy. I'll handle any editing. Thank you.

**Review by chri2547 — COMMENTED** [review]
The docs look fine to me. Awaiting engr input. Thank you for the contribution!

**Comment by Symbianx** [comment]
Please let me know if there's anything else I need to do! :)

**Comment by daniellee** [comment]
Think this is one for @grafana/observability-traces-and-profiling

**Comment by aocenas** [comment]
Sorry to just pass this to the next team but I think this is more of a @grafana/backend-platform thing probably. We don't really do much with the tracing of Grafana itself at this moment.

**Comment by Symbianx** [comment]
@chri2547 The PR was marked as stale, please let me know if there's anything else I should do.

**Review by chri2547 — APPROVED** [review]

**Review by sakjur — APPROVED** [review]
Much too late review: This looks like an improvement to status quo that we could merge as-is, but I also think we can improve on this a little bit more and then merge, see separate comments.

File: docs/sources/setup-grafana/set-up-grafana-monitoring.md [hunk 1]
```diff
@@ -15,13 +15,15 @@ weight: 800
...
 
-Grafana can emit Jaeger traces for its HTTP API endpoints and propagate Jaeger trace information to data sources.
+Grafana can emit Jaeger/OTLP traces for its HTTP API endpoints and propagate Jaeger/w3c trace information to data sources.
 All HTTP endpoints are logged evenly (annotations, dashboard, tags, and so on).
 When a trace ID is propagated, it is reported with operation 'HTTP /datasources/proxy/:id/\*'.
 
-Refer to [Configuration]({{< relref "configure-grafana/#tracingjaeger" >}}) for information about enabling Jaeger tracing.
+Refer to [Configuration]({{< relref "configure-grafana/#tracingopentelemetryjaeger" >}}) for information about enabling Jaeger tracing.
+
+Refer to [Configuration]({{< relref "configure-grafana/#tracingopentelemetryotlp" >}}) for information about enabling OTLP tracing.
```
[the remark this answers is not in the document]
Comment (Symbianx) [reply docs/sources/setup-grafana/set-up-grafana-monitoring.md:26]: This makes sense, if it's just a reference let's keep that expectation from the begining 👍

**Pull request merged**